### PR TITLE
Info Box component

### PR DESCRIPTION
**Pull request by luizzaly**
## Description
Info Box Component, Cancel Button should close Info Box, consists out of three slots: header, body and actions
needed to add color-info to Base Button

## Screenshot

Screenshot 2020-03-17 at 12 25 34

### CODE REVIEW

File: src/components/base/BaseButton.scss [hunk 1]
```diff
@@ -43,6 +43,13 @@ button {
 		&:hover {
 			--button-background: var(--color-danger--dark);
 		}
+	}
+		&.is-info {
+		--button-background: var(--color-info);
+		--button-text-color: var(--color-white);
```
Comment (adrianjost) [inline src/components/base/BaseButton.scss]:
```suggestion
			--button-background: var(--color-info);
			--button-text-color: var(--color-white);
```

File: src/components/base/BaseButton.vue [hunk 2]
```diff
@@ -50,6 +50,8 @@ export default {
 					"primary icon outline",
 					"primary text icon",
 					"primary outline",
+					"info",
+					"info outline",
```
Comment (adrianjost) [inline src/components/base/BaseButton.vue]: - [ ] please also add all the other possible design types for `info` like `info icon` and so on
- [ ] You also need to add those new designs to the storybook knobs

File: src/components/molecules/InfoBox.stories.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+	})
+	.add("InfoBox", () => ({
+		components: { InfoBox, BaseButton, BaseInput },
+			data: () => ({
+			active: true,
+			header: text("header", "Einverständniserklärung"),
+			body: text(
+				"content",
+				"Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet."
```
Comment (adrianjost) [inline src/components/molecules/InfoBox.stories.js]: You can use faker to auto generate some cool texts.

```
import * as faker from "faker";
const text = faker.lorem.paragraph()
```
https://www.npmjs.com/package/faker

Comment (adrianjost) [reply src/components/molecules/InfoBox.stories.js]: - [ ] screenshots for screenshot tests are missing. (`npm run dev:storybook` und parallel `npm run test:screenshot:jest -- -u`)

**Comment by dominikbruechner** [comment]
muss noch gelinted werden
screenshot tests failen

**Comment by adrianjost** [comment]
@schul-cloud-bot update screenshots

**Pull request merged**